Mar 28 22:35:01 k8s-node-master kubelet[3234]: I0328 22:35:01.978368    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:01 k8s-node-master containerd[1936]: time="2025-03-28T22:35:01.983416451Z" level=info msg="RunPodSandbox for &PodSandboxMetadata{Name:etcd-k8s-node-master,Uid:f05bda3a2f28b4df60c28cfabc1b1940,Namespace:kube-system,Attempt:4,}"
Mar 28 22:35:02 k8s-node-master containerd[1936]: time="2025-03-28T22:35:02.081385766Z" level=info msg="RunPodSandbox for &PodSandboxMetadata{Name:etcd-k8s-node-master,Uid:f05bda3a2f28b4df60c28cfabc1b1940,Namespace:kube-system,Attempt:4,} returns sandbox id \"63d9dccb6a2b348628d363ea668dac9144991c196b12961ba8db572facfd7419\""
Mar 28 22:35:02 k8s-node-master kubelet[3234]: E0328 22:35:02.082258    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:35:02 k8s-node-master kubelet[3234]: E0328 22:35:02.981027    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:35:02 k8s-node-master kubelet[3234]: I0328 22:35:02.981467    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:02 k8s-node-master kubelet[3234]: I0328 22:35:02.983440    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:03 k8s-node-master kubelet[3234]: I0328 22:35:03.987558    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:10 k8s-node-master kubelet[3234]: I0328 22:35:10.792543    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:10 k8s-node-master kubelet[3234]: E0328 22:35:10.965710    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:35:20 k8s-node-master kubelet[3234]: I0328 22:35:20.792600    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:21 k8s-node-master containerd[1936]: time="2025-03-28T22:35:21.795130531Z" level=info msg="CreateContainer within sandbox \"63d9dccb6a2b348628d363ea668dac9144991c196b12961ba8db572facfd7419\" for container &ContainerMetadata{Name:etcd,Attempt:4,}"
Mar 28 22:35:21 k8s-node-master containerd[1936]: time="2025-03-28T22:35:21.835004995Z" level=info msg="CreateContainer within sandbox \"63d9dccb6a2b348628d363ea668dac9144991c196b12961ba8db572facfd7419\" for &ContainerMetadata{Name:etcd,Attempt:4,} returns container id \"48c5d7041deddc435daf6ec3f37014b9014f7b4475652b5b6723ad96de91640b\""
Mar 28 22:35:22 k8s-node-master kubelet[3234]: I0328 22:35:22.029159    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:30 k8s-node-master kubelet[3234]: I0328 22:35:30.792878    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:40 k8s-node-master kubelet[3234]: I0328 22:35:40.792766    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:40 k8s-node-master kubelet[3234]: I0328 22:35:40.940298    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:40 k8s-node-master kubelet[3234]: I0328 22:35:40.941445    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:42 k8s-node-master kubelet[3234]: I0328 22:35:42.076141    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:42 k8s-node-master containerd[1936]: time="2025-03-28T22:35:42.077369928Z" level=info msg="RunPodSandbox for &PodSandboxMetadata{Name:etcd-k8s-node-master,Uid:f05bda3a2f28b4df60c28cfabc1b1940,Namespace:kube-system,Attempt:5,}"
Mar 28 22:35:42 k8s-node-master containerd[1936]: time="2025-03-28T22:35:42.165802797Z" level=info msg="RunPodSandbox for &PodSandboxMetadata{Name:etcd-k8s-node-master,Uid:f05bda3a2f28b4df60c28cfabc1b1940,Namespace:kube-system,Attempt:5,} returns sandbox id \"28bde44ab03ed18ab8b3d5caa3dc18f055b17e7c7abe8136260ecc14d7c96d2d\""
Mar 28 22:35:42 k8s-node-master kubelet[3234]: E0328 22:35:42.166531    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 40s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:35:43 k8s-node-master kubelet[3234]: E0328 22:35:43.079543    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 40s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:35:43 k8s-node-master kubelet[3234]: I0328 22:35:43.079684    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:44 k8s-node-master kubelet[3234]: E0328 22:35:44.081741    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 40s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:35:50 k8s-node-master kubelet[3234]: I0328 22:35:50.792951    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:35:50 k8s-node-master kubelet[3234]: E0328 22:35:50.938910    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 40s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:36:00 k8s-node-master kubelet[3234]: I0328 22:36:00.793456    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:36:06 k8s-node-master kubelet[3234]: E0328 22:36:06.793363    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 40s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:36:10 k8s-node-master kubelet[3234]: I0328 22:36:10.793173    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:36:20 k8s-node-master kubelet[3234]: I0328 22:36:20.793195    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:36:21 k8s-node-master containerd[1936]: time="2025-03-28T22:36:21.795446125Z" level=info msg="CreateContainer within sandbox \"28bde44ab03ed18ab8b3d5caa3dc18f055b17e7c7abe8136260ecc14d7c96d2d\" for container &ContainerMetadata{Name:etcd,Attempt:5,}"
Mar 28 22:36:21 k8s-node-master containerd[1936]: time="2025-03-28T22:36:21.812884030Z" level=info msg="CreateContainer within sandbox \"28bde44ab03ed18ab8b3d5caa3dc18f055b17e7c7abe8136260ecc14d7c96d2d\" for &ContainerMetadata{Name:etcd,Attempt:5,} returns container id \"99e18445944d539bf2fad9391f8d17fe24512b912f5b0b9143488d3dd85e2f8b\""
Mar 28 22:36:22 k8s-node-master kubelet[3234]: I0328 22:36:22.167205    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:36:30 k8s-node-master kubelet[3234]: I0328 22:36:30.793141    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:36:31 k8s-node-master kubelet[3234]: I0328 22:36:31.936830    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="pods \"etcd-k8s-node-master\" is forbidden: User \"system:node:k8s-node-master\" cannot get resource \"pods\" in API group \"\" in the namespace \"kube-system\": no relationship found between node 'k8s-node-master' and this object"
Mar 28 22:40:32 k8s-node-master containerd[1936]: time="2025-03-28T22:40:32.725712599Z" level=info msg="RunPodSandbox for &PodSandboxMetadata{Name:etcd-k8s-node-master,Uid:f05bda3a2f28b4df60c28cfabc1b1940,Namespace:kube-system,Attempt:6,}"
Mar 28 22:40:32 k8s-node-master containerd[1936]: time="2025-03-28T22:40:32.794742304Z" level=info msg="RunPodSandbox for &PodSandboxMetadata{Name:etcd-k8s-node-master,Uid:f05bda3a2f28b4df60c28cfabc1b1940,Namespace:kube-system,Attempt:6,} returns sandbox id \"01097b10a59acfd2f1e7b34a4929cfb58f981cde98ece68d5dab3830daf8da3d\""
Mar 28 22:40:32 k8s-node-master kubelet[3234]: E0328 22:40:32.795530    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:40:33 k8s-node-master kubelet[3234]: E0328 22:40:33.724353    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:40:34 k8s-node-master kubelet[3234]: E0328 22:40:34.726640    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:40:40 k8s-node-master kubelet[3234]: E0328 22:40:40.962813    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:40:56 k8s-node-master kubelet[3234]: E0328 22:40:56.793289    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:41:05 k8s-node-master kubelet[3234]: E0328 22:41:05.946229    3234 event.go:359] "Server rejected event (will not retry!)" err="Timeout: request did not complete within requested timeout - context deadline exceeded" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311780621bdbaa  kube-system   1281 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:Unhealthy,Message:Readiness probe failed: Get \"http://127.0.0.1:2381/readyz\": dial tcp 127.0.0.1:2381: connect: connection refused,Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:19:30 +0000 UTC,LastTimestamp:2025-03-28 22:40:31.943344257 +0000 UTC m=+1291.217463830,Count:5,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:41:08 k8s-node-master kubelet[3234]: E0328 22:41:08.793152    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:41:22 k8s-node-master kubelet[3234]: E0328 22:41:22.818327    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:41:32 k8s-node-master kubelet[3234]: I0328 22:41:32.723898    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="the server was unable to return a response in the time allotted, but may still be processing the request (get pods etcd-k8s-node-master)"
Mar 28 22:41:35 k8s-node-master kubelet[3234]: E0328 22:41:35.818138    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:41:39 k8s-node-master kubelet[3234]: E0328 22:41:39.949238    3234 event.go:359] "Server rejected event (will not retry!)" err="Timeout: request did not complete within requested timeout - context deadline exceeded" event="&Event{ObjectMeta:{etcd-k8s-node-master.183117809947cccd  kube-system   1282 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:,},Reason:SandboxChanged,Message:Pod sandbox changed, it will be killed and re-created.,Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:19:31 +0000 UTC,LastTimestamp:2025-03-28 22:40:32.720049677 +0000 UTC m=+1291.994169250,Count:6,Type:Normal,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:41:47 k8s-node-master kubelet[3234]: E0328 22:41:47.793233    3234 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"etcd\" with CrashLoopBackOff: \"back-off 1m20s restarting failed container=etcd pod=etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940)\"" pod="kube-system/etcd-k8s-node-master" podUID="f05bda3a2f28b4df60c28cfabc1b1940"
Mar 28 22:42:01 k8s-node-master containerd[1936]: time="2025-03-28T22:42:01.798028655Z" level=info msg="CreateContainer within sandbox \"01097b10a59acfd2f1e7b34a4929cfb58f981cde98ece68d5dab3830daf8da3d\" for container &ContainerMetadata{Name:etcd,Attempt:6,}"
Mar 28 22:42:01 k8s-node-master containerd[1936]: time="2025-03-28T22:42:01.812620560Z" level=info msg="CreateContainer within sandbox \"01097b10a59acfd2f1e7b34a4929cfb58f981cde98ece68d5dab3830daf8da3d\" for &ContainerMetadata{Name:etcd,Attempt:6,} returns container id \"a853bddde7d0ad5e6b5e085430fdc31b1686d05244b8d87960e6a93d3b36da72\""
Mar 28 22:42:13 k8s-node-master kubelet[3234]: E0328 22:42:13.952052    3234 event.go:359] "Server rejected event (will not retry!)" err="Timeout: request did not complete within requested timeout - context deadline exceeded" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:32.79550515 +0000 UTC m=+1292.069624723,Count:13,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:42:13 k8s-node-master kubelet[3234]: E0328 22:42:13.952630    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:42:19 k8s-node-master kubelet[3234]: I0328 22:42:19.480869    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:42:19 k8s-node-master kubelet[3234]: I0328 22:42:19.481354    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:42:19 k8s-node-master kubelet[3234]: I0328 22:42:19.481848    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:42:20 k8s-node-master kubelet[3234]: I0328 22:42:20.792910    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:42:20 k8s-node-master kubelet[3234]: I0328 22:42:20.940311    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:42:20 k8s-node-master kubelet[3234]: I0328 22:42:20.941491    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="Get \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/pods/etcd-k8s-node-master\": dial tcp 192.168.122.180:6443: connect: connection refused"
Mar 28 22:42:22 k8s-node-master kubelet[3234]: E0328 22:42:22.889048    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:42:24 k8s-node-master kubelet[3234]: I0328 22:42:24.336963    3234 status_manager.go:890] "Failed to get status for pod" podUID="f05bda3a2f28b4df60c28cfabc1b1940" pod="kube-system/etcd-k8s-node-master" err="pods \"etcd-k8s-node-master\" is forbidden: User \"system:node:k8s-node-master\" cannot get resource \"pods\" in API group \"\" in the namespace \"kube-system\": no relationship found between node 'k8s-node-master' and this object"
Mar 28 22:42:32 k8s-node-master kubelet[3234]: E0328 22:42:32.890432    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:42:42 k8s-node-master kubelet[3234]: E0328 22:42:42.891723    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:42:52 k8s-node-master kubelet[3234]: E0328 22:42:52.893522    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:43:02 k8s-node-master kubelet[3234]: E0328 22:43:02.895071    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:43:12 k8s-node-master kubelet[3234]: E0328 22:43:12.896225    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:43:22 k8s-node-master kubelet[3234]: E0328 22:43:22.898040    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:43:32 k8s-node-master kubelet[3234]: E0328 22:43:32.899486    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
Mar 28 22:43:42 k8s-node-master kubelet[3234]: E0328 22:43:42.902313    3234 event.go:368] "Unable to write event (may retry after sleeping)" err="Patch \"https://192.168.122.180:6443/api/v1/namespaces/kube-system/events/etcd-k8s-node-master.18311853567e37d2\": dial tcp 192.168.122.180:6443: connect: connection refused" event="&Event{ObjectMeta:{etcd-k8s-node-master.18311853567e37d2  kube-system   1287 0 0001-01-01 00:00:00 +0000 UTC <nil> <nil> map[] map[] [] [] []},InvolvedObject:ObjectReference{Kind:Pod,Namespace:kube-system,Name:etcd-k8s-node-master,UID:f05bda3a2f28b4df60c28cfabc1b1940,APIVersion:v1,ResourceVersion:,FieldPath:spec.containers{etcd},},Reason:BackOff,Message:Back-off restarting failed container etcd in pod etcd-k8s-node-master_kube-system(f05bda3a2f28b4df60c28cfabc1b1940),Source:EventSource{Component:kubelet,Host:k8s-node-master,},FirstTimestamp:2025-03-28 22:34:36 +0000 UTC,LastTimestamp:2025-03-28 22:40:33.72433366 +0000 UTC m=+1292.998453233,Count:14,Type:Warning,EventTime:0001-01-01 00:00:00 +0000 UTC,Series:nil,Action:,Related:nil,ReportingController:kubelet,ReportingInstance:k8s-node-master,}"
